rascal-vscode-extension/src/test/vscode-suite/ide.test.ts

```diff
@@ -152,5 +153,21 @@ describe('IDE', function () {
         await driver.actions().doubleClick(mainItem!).perform();
         await driver.wait(async ()=> (await editor.getCoordinates())[0] === 5, Delays.normal, "Cursor should have moved to line that contains the println function");
     });
+
+    it ("rename works", async() => {
+        const editor = await ide.openModule(TestWorkspace.libFile);
+        await editor.moveCursor(7, 15);
+
+        await bench.executeCommand("Rename Symbol");
+        const renameBox = await driver.wait(async () => await bench.getEditorView().findElement(By.className("rename-input")), Delays.fast, "Rename box should appear");
```

Comment:
maybe we can do something that reuses the same functions that are present in `util.ts`?

Reply:
Had a look, but I don't see anything related to finding arbitrary editor elements there. I can refactor once we write more related tests (e.g. cross-module and/or other refactorings).

Reply:
I meant that for example the `hasElement` function looks to do exactly what this line is doing.

Reply:
Missed that completely. Fixed it.